### PR TITLE
160 GitHub actions

### PR DESCRIPTION
**Pull request by ahjyrkia**
Cypress tests moved to be ran with github actions

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -3,9 +3,9 @@
 REACT_APP_API_URL=http://localhost:4000
 REACT_APP_JORE_API_URL=https://dev.kartat.hsl.fi/jore/graphql
 REACT_APP_CLIENT_ID=7833861618225795
-REACT_APP_CLIENT_SECRET=
```
Comment (Pesonet1) [inline .env:6]: Why these area removed only from .env file? Should these changes be propagated to .dev, .stage & .prod env-files?

File: constants.js [hunk 2]
```diff
@@ -1,35 +0,0 @@
-const { mapValues, orderBy } = require('lodash');
```
Comment (Pesonet1) [inline constants.js:1]: Why this file is removed? CLIENT_SECRET is still referenced in .dev, .stage & .production env-files?

File: cypress/support/commands.js [hunk 3]
```diff
@@ -2,11 +2,10 @@ Cypress.Commands.add('hslLogin', () => {
   const AUTH_URI = 'https://hslid-uat.cinfra.fi/openid/token';
   const AUTH_SCOPE = 'email https://oneportal.trivore.com/scope/groups.readonly';
 
-  const HSL_TESTING_HSLID_USERNAME = process.env.HSL_TESTING_HSLID_USERNAME || '';
```
Comment (Pesonet1) [inline cypress/support/commands.js:5]: If these are not used anymore in deployment, these keys reference should be removed from env-files

File: .env [hunk 4]
```diff
@@ -3,9 +3,9 @@
...
 REACT_APP_REDIRECT_URI=http://localhost:3000
-HSL_TESTING_HSLID_USERNAME=
-HSL_TESTING_HSLID_PASSWORD=
-DOMAINS_ALLOWED_TO_GENERATE=
-REACT_APP_NAMESPACE=hsl-kartta
```
Comment (Pesonet1) [inline .env:11]: This value (REACT_APP_NAMESPACE) is not set in stage env-file

**Review by Pesonet1 — CHANGES_REQUESTED** [review]

File: .env [hunk 5]
```diff
@@ -1,11 +1,12 @@
-# Shared environment variables. Add env variables specific to your environment in .env.development.local
+# Shared environment variables. Add env variables specific to your environment in .env
 
 REACT_APP_API_URL=http://localhost:4000
 REACT_APP_JORE_API_URL=https://dev.kartat.hsl.fi/jore/graphql
 REACT_APP_CLIENT_ID=7833861618225795
-REACT_APP_CLIENT_SECRET=
 REACT_APP_REDIRECT_URI=http://localhost:3000
-HSL_TESTING_HSLID_USERNAME=
```
Comment (Pesonet1) [inline .env:8]: These secrets should be removed after applying this change, so that no hanging secrets exist.

**Review by Pesonet1 — CHANGES_REQUESTED** [review]

File: .env (hunk 4 above)
Comment (ahjyrkia) [reply .env:11]: Done

File: .env (hunk 1 above)
Comment (ahjyrkia) [reply .env:6]: Removed from all envs

File: .env (hunk 5 above)
Comment (ahjyrkia) [reply .env:8]: Removed from all envs

File: cypress/support/commands.js (hunk 3 above)
Comment (ahjyrkia) [reply cypress/support/commands.js:5]: Removed

File: constants.js (hunk 2 above)
Comment (ahjyrkia) [reply constants.js:1]: Removed from env files

**Pull request merged**